### PR TITLE
Update item categories and URL Defaults

### PR DESCRIPTION
**Pull request by andykim**
## Description

Update item categories and URL Defaults

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot

`put screenshot here if there are any ui changes`

### CODE REVIEW

**Review by rendegosling — APPROVED** [review]

File: Xendit/M2Invoice/Controller/Checkout/Invoice.php [hunk 1]
```diff
@@ -87,10 +87,10 @@ private function getApiRequestData($order)
             $categoryName = implode(', ', $categories);
             $item['reference_id'] = $product->getId();
             $item['name'] = $product->getName();
-            $item['category'] = $categoryName ?: 'Uncategorized';
+            $item['category'] = $categoryName ?: 'N/A';
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/Controller/Checkout/Invoice.php:90]: can put small letter 'n/a'?

File: Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php [hunk 2]
```diff
@@ -83,10 +83,10 @@ public function execute()
                     $categoryName = implode(', ', $categories);
                     $item['reference_id'] = $product->getId();
                     $item['name'] = $product->getName();
-                    $item['category'] = $categoryName ?: 'Uncategorized';
+                    $item['category'] = $categoryName ?: 'N/A';
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php:86]: this too

**Review by candrasaputra — COMMENTED** [review]
LGTM, other then irene comment

**Review by candrasaputra — APPROVED** [review]

File: Xendit/M2Invoice/Controller/Checkout/Invoice.php (hunk 1 above)
Comment (andykim) [reply Xendit/M2Invoice/Controller/Checkout/Invoice.php:90]: @IreneGohtami yep. Let me use the lower case for all `N/A`.

Comment (andykim) [reply Xendit/M2Invoice/Controller/Checkout/Invoice.php]: Updated @IreneGohtami . plz check

**Pull request merged**